editor/editor_help.cpp

```diff
@@ -2581,19 +2581,19 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 			const String codeblock_text = bbcode.substr(brk_end + 1, end_pos - (brk_end + 1)).strip_edges();
 
 			String codeblock_copy_text = codeblock_text;
-			if (using_tab_indent) {
-				// Replace the code block's space indentation with tabs.
+			if (!using_tab_indent) {
```

Comment:
Nitpicking, but I would prefer to invert the condition to avoid negation:

```suggestion
			if (using_space_indent) {
```

https://github.com/godotengine/godot/blob/ccb1cb4845c3a70c5e54d1690bdf14024095ab35/editor/editor_help.cpp#L2326

```c++
const bool using_space_indent = int(EDITOR_GET("text_editor/behavior/indent/type")) == 1;
```